src/Controller/BugController.php

```diff
@@ -50,13 +51,22 @@ public function search(Request $request, BugService $service): Response
         ]);
     }
 
-    #[Route(path: '/create', name: 'bug_new', methods: ['GET', 'POST'])]
-    public function new(Request $request, BugService $service): Response
+    #[Route(path: '/init', name: 'bug_init', methods: ['GET', 'POST'])]
+    public function init(Request $request, UserRequestManager $manager): Response
+    {
+        $bug = new Bug($request->headers->get('User-Agent', ''));
+        $manager->create($bug);
```

Comment:
Maybee, we could have somethinf more like 
```
$bug = $manager->create($request->headers->get('User-Agent', ''))
```

Reply:
I did it this way to have $manager->create() identical for both feature and bug.